static/directives/repository-events-table.html

```diff
@@ -6,7 +6,7 @@
       <div class="heading-controls hidden-sm hidden-xs">
         <a href="/repository/{{ repository.namespace }}/{{ repository.name }}/create-notification"
            class="btn btn-primary" ng-show="repository.can_write && !inReadOnlyMode">
-          <i class="fa fa-plus"></i> Create Notification
+           <span class="create-notification-btn"><i class="fa fa-plus"></i> Create Notification</span>
```

Comment:
Fixes this CSS issue when on-prem (blue link overriding button text color):
![Screenshot_20191122_162121](https://user-images.githubusercontent.com/11700385/69461287-31a42680-0d44-11ea-9afb-25bef4624067.png)